src/main/java/com/mattermost/integration/figma/api/mm/bindings/BindingServiceImpl.java

```diff
@@ -0,0 +1,72 @@
+package com.mattermost.integration.figma.api.mm.bindings;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.mattermost.integration.figma.api.mm.kv.KVService;
+import com.mattermost.integration.figma.api.mm.user.MMUserService;
+import com.mattermost.integration.figma.input.mm.binding.Binding;
+import com.mattermost.integration.figma.input.mm.binding.BindingsDTO;
+import com.mattermost.integration.figma.input.mm.binding.Command;
+import com.mattermost.integration.figma.input.mm.user.MMUser;
+import com.mattermost.integration.figma.input.oauth.InputPayload;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Objects;
+
+@Service
+@Slf4j
+public class BindingServiceImpl implements BindingService {
+    private static final String DEFAULT_BINDINGS_PATH = "src\\main\\resources\\static\\bindings.json";
```

Comment:
will this path  work with java --jar command ? Inside jar/zip we don't have this folder structure